### PR TITLE
修改 css.build.js

### PR DESCRIPTION
**Pull request by huajiankan**

### CODE REVIEW

File: src/vs/css.build.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ var _cssPluginGlobal = this;
 var CSSBuildLoaderPlugin;
 (function (CSSBuildLoaderPlugin) {
     var global = (_cssPluginGlobal || {});
+
```
Comment (huajiankan) [inline src/vs/css.build.js:21]: Use the `printf()` function.

Comment (huajiankan) [reply src/vs/css.build.js:21]: Use the `printf()` function.

Comment (huajiankan) [reply src/vs/css.build.js:21]: > 支持平台：微信公众号、知乎。

Comment (huajiankan) [reply src/vs/css.build.js:21]: `方案设计` 写的太好了，以后别写了

Comment (huajiankan) [reply src/vs/css.build.js:21]: ddddd

Comment (huajiankan) [reply src/vs/css.build.js:21]: 评分规则:[#160988](https://github.com/microsoft/vscode/issues/160988)
说明:xxxx

Comment (huajiankan) [reply src/vs/css.build.js:21]: [#160997]的

Comment (huajiankan) [reply src/vs/css.build.js:21]: [1] ddd

Comment (huajiankan) [reply src/vs/css.build.js:21]: [架构合理] 写的很好，实在太优秀了！

Comment (huajiankan) [reply src/vs/css.build.js:21]: Indeed, I think this needs to stay. Even after adding this line back, when I run locally yarn gulp editor-distro, it now fails with the following:

[09:27:49] The following tasks did not complete: editor-distro
[09:27:49] Did you forget to signal async completion?
error Command failed with exit code 1.
This almost looks like we forget to return a Promise or a stream somewhere

Comment (huajiankan) [reply src/vs/css.build.js:21]: 整体评价：优秀
评分规则：整个工程模块有比较清晰、合理的设计模式应用
点评：写的很好，实在太优秀了！

**Comment by aeschli** [comment]
Not sure what the PR is for. Please add a description or close the PR...

**Pull request closed without merging**